pkg/snapshots/snapshots.go

```diff
@@ -0,0 +1,135 @@
+package snapshots
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/aybabtme/godotto/internal/godojs"
+	"github.com/aybabtme/godotto/internal/ottoutil"
+	"github.com/aybabtme/godotto/pkg/extra/do/cloud"
+	"github.com/aybabtme/godotto/pkg/extra/do/cloud/snapshots"
+	"github.com/robertkrimen/otto"
+)
+
+var q = otto.Value{}
+
+func Apply(ctx context.Context, vm *otto.Otto, client cloud.Client) (otto.Value, error) {
+	root, err := vm.Object(`({})`)
+	if err != nil {
+		return q, err
+	}
+
+	svc := snapshotsSvc{
+		ctx: ctx,
+		svc: client.Snapshots(),
+	}
+
+	for _, applier := range []struct {
+		Name   string
+		Method interface{}
+	}{
+		{"get", svc.get},
+		{"list", svc.list},
+		{"listDroplet", svc.listDroplet},
+		{"listVolume", svc.listVolume},
```

Comment:
For uniformity, could we change this to `"list_volume"` please? Just the string, not the function definition itself. :)